user-manual/transfer/import-metadata.rst

```diff
@@ -93,15 +93,15 @@ for creating a metadata.csv file that will apply metadata to individual objects.
 #. Create a CSV file called metadata.csv. The example below can be used as
    a template for Dublin Core metadata. The filename column must come first and
    the filename path must always start with ``objects/``. Note that if you add
-   column headings that are not in the ``dc.element`` format, Archivematica will
-   not indicate that the metadata is Dublin Core.
+   column headings that are not in the ``dc.element`` or ``dcterms.element``
+   formats, Archivematica will not indicate that the metadata is Dublin Core.
 
    .. csv-table::
       :file: _csv/individual-metadata.csv
       :header-rows: 1
 
-   Note that empty columns (i.e. ``dc.contributor``) were left in to demonstrate
-   the full range of possible Dublin Core values. You do not need to keep empty
+   Note that empty columns (e.g. ``dc.contributor``) were left in to demonstrate
```

Comment:
Also a good catch, I'm really trying to break myself of this weird habit :joy:

Reply:
oh you mean i.e. vs. e.g.! 🤣  Yes, I Google this EVERY SINGLE TIME. You are not alone. It was only because I had just Googled it that I caught it.